.gitignore

```diff
@@ -2,3 +2,5 @@
 meeting_minutes/.DS_Store
 .DS_Store
 *_strict_schema.json
+official-cpe-dictionary_v2.3.*
```

Comment:
I can imagine a single `official-cpe-dictionary_v2*` would do but I am `+0` on just keeping your version - after all it is only a git ignore file.

Reply:
I tend to be a bit more specific in `.gitignore` as it helps me to avoid the "works on my machine". But I'm happy with both variants...